### DISCUSSION TITLE
[Question][Hw4] initial seeds

### DISCUSSION THREAD

**Discussion by esk223** [prosyslab-classroom/cs348-information-security#558] [Q&A]
I want to maintain the number of seeds set, and I guess the current code has one (in the string) or two (in JSON) initial seed(s).
During the evaluation, are the initial seeds same as the current ones?
If not, can you tell us the number of initial seeds?
(Because limiting the size of seeds set to 10 when the number of initial seeds is 20, I think the algorithm is bad.)

**Comment by kimjungwow** [comment]
As noticed in https://docs.google.com/document/d/1zIM83xkeyy8lBDJa1C6QVWM2rqUFxoyX9FIHqbQTSCg/edit,
**the same seed inputs will be used for evaluation.**

**Comment by esk223** [comment]
Oh I missed that. Thank you!
